### DISCUSSION TITLE
Drag&Drop management for folders and items

### DISCUSSION THREAD

**Discussion by Kedryn** [dani-garcia/vaultwarden#3445] [Ideas]
### Subject of the issue
Web interface of Vaultwarden

### Deployment environment

* vaultwarden version:  0.19.3

* Install method: as Home Assistant addon

* Clients used: 
web

* Reverse proxy and version: 
no

* MySQL/MariaDB or PostgreSQL version: 
MySQL

Mine is just a request for enhancement
I installed this docker in my Home Assistant, willing to move all my Keepas data to Vaultwarden
i esported from keepass, imported in VW, ready to reorganize file and folder to fit VW way to manage them.

and i found out that not only there is no drag&drop but moving items from a folder to another, when you have hundred of them, is an overwhelming task. I tried, but after 30 minutes i gave up.

so, my request is that: please make the management of a fairly large library, a simpler task, either than implementing a drag&drop, or at least letting select a folder and move it with all it's content in some way.

Thanks

**Comment by stefan0xC** [comment]
What version is 0.19.3? I don't think that's a vaultwarden version. Which web-vault version is that?

Regarding your feature request (which should be directed to Bitwarden): you can filter by folder, select all items and use the move selected option to move multiple items to another folder.

https://user-images.githubusercontent.com/509385/231865127-a4156464-fb5f-4975-a13c-20331a16d49b.mp4

**Reply by Kedryn** [reply]
Sorry, maybe the version was from the HA addon and not the VaultWarden one, but anyway is the latest one available, there is really no need for the specific version as the web interface is the same.

Yes, you can select/move, ITEMS, but when you have hundred of organized items, using that way is a pain in the a**. I didn't say it's impossibile, i did say us unpractical, and if you do the slightest error, it may be better starting from scratch than working to recover it.

What i'm saying is that, coming from other password managers, the *warden interface is nearly like working directly on the DB. It's not user friendly at all. It's fine if all the data is already organized, but reaching that point is a daunting task.

The UI needs a lot of love.

**Reply by BlackDex** [reply]
I suggest to report/ suggest UI improvements at Bitwarden. This project doesn't maintain the UI's or any clients for that matter.
https://community.bitwarden.com/c/feature-requests/5/none
